platform/windows/os_windows.cpp

```diff
@@ -1437,7 +1437,7 @@ DWRITE_FONT_STRETCH OS_Windows::_stretch_to_dw(int p_stretch) const {
 }
 
 Vector<String> OS_Windows::get_system_font_path_for_text(const String &p_font_name, const String &p_text, const String &p_locale, const String &p_script, int p_weight, int p_stretch, bool p_italic) const {
-	if (!dwrite2_init) {
+	if (!dwrite2_init || !TextServerManager::get_singleton()) {
```

Comment:
It'd be great that we could make it explicit here the reason for the null check. Since we can't query the current initialization level, a comment should do the trick.